jiant/utils/utils.py

```diff
@@ -33,6 +33,20 @@
 _MOSES_DETOKENIZER = MosesDetokenizer()
 
 
+def get_output_attribute(out, attribute_name):
+    if torch.cuda.device_count() > 1:
+        return out[attribute_name].sum()
+    else:
+        return out[attribute_name]
+
+
+def get_model_attribute(model, attribute_name):
```

Comment:
This one isn't completely clear. Add a docstring explaining what this does/fixes.

Reply:
I'm still not sure I understand. Add a short explanation for why this is necessary...